src/main/java/io/logz/guice/jersey/JerseyModule.java

```diff
@@ -5,22 +5,30 @@
 import com.google.inject.Provider;
 import com.google.inject.servlet.ServletModule;
 import io.logz.guice.jersey.configuration.JerseyConfiguration;
+import org.eclipse.jetty.server.Server;
 
 import java.util.Objects;
+import java.util.function.Consumer;
 
 public class JerseyModule extends AbstractModule {
 
     private final JerseyConfiguration jerseyConfiguration;
+    private Consumer<Server> jettyConfigurer;
 
     public JerseyModule(JerseyConfiguration jerseyConfiguration) {
+        this(jerseyConfiguration, server -> {});
+    }
+
+    public JerseyModule(JerseyConfiguration jerseyConfiguration, Consumer<Server> jettyConfigurer) {
```

Comment:
In order to let our users the possibility to configure the Jetty server and control the thread poll let's add the option to specify `JettyServerCreator` with default implementation of one that returns `new Server()` when using the old constructor.

Create interface named `JettyServerCreator` with a method `create`